tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "strictNullChecks": true,
     "strictFunctionTypes": true,
     "typeRoots": ["node_modules/@types"],
-    "jsx": "react"
+    "jsx": "react",
+    "types": ["node"]
```

Comment:
Don't need this. Should remove it.

```suggestion
```

Reply:
I'm pretty sure without it the [build script broke](https://app.circleci.com/pipelines/github/dequelabs/cauldron/180/workflows/7fcfcafd-4583-48af-8d3c-8b52021253cb/jobs/736) but I'll remove it and give it a try 🤷‍♂️ 

```
[!] (plugin typescript) Error: @rollup/plugin-typescript TS2580: Cannot find name 'process'. Do you need to install type definitions for node? Try `npm i @types/node`.
src/components/Icon/index.tsx (21:9)
```

Reply:
Yep. Removing `"types": ["node"]` broke the build: https://app.circleci.com/pipelines/github/dequelabs/cauldron/183/workflows/933ecaeb-99a4-47cc-941a-730939dcf718/jobs/747

Reply:
Strange.. might have something to do with setting `typeRoots` (which is unnecessary most of the time), or using Babel to compile TypeScript. It's probably worth looking into this in the future. Not going to block this PR for that tho.